bin/create-digests.py

```diff
@@ -117,11 +117,175 @@ def collect_digests(digests, subdir, exclude=[]):
                         digests[filepath] = {}
                     digests[filepath] = digest.hexdigest()
 
+def add_totals(digests, total_key='total_contributions', total_group_key=None, filepath='build/_data/totals.json'):
```

Comment:
Probably good to have a comment here about what is going on.

Reply:
done